common/app/conf/switches/DiscussionSwitches.scala

```diff
@@ -43,4 +43,14 @@ trait DiscussionSwitches {
     exposeClientSide = true
   )
 
+  val DiscussionStopTrafficFromDotcom = Switch(
```

Comment:
From a frontend perspective I prefer to see this switch as "disabling discussion". Why not naming it `DisableDiscussionSwitch` with a description that mention it removes the discussion UI as well as blocking all traffic from frontend to DAPI?